packages/devtools_app/lib/src/charts/flame_chart_canvas.dart

```diff
@@ -378,13 +378,8 @@ abstract class FlameChartCanvas<T> extends FlameChart {
 }
 
 class FlameChartRow {
-  const FlameChartRow({
-    @required this.nodes,
-    @required this.index,
-  });
-
+  FlameChartRow({@required this.nodes});
```

Comment:
is nodes always getting passed in as an empty array? If so, simplify by removing the nodes named parameter.